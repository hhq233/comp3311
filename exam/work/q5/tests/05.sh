python3 q5.py "san diego"
