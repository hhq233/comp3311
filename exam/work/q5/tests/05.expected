AleSmith Brewing Co, who brewed 7 beers in 2019 and 2020
Ballast Point Brewing Co, who brewed 3 beers in 2018 and 2019
Belching Beaver Brewery, who brewed 4 beers in 2019 and 2020
Karl Strauss Brewing Company, who brewed 1 beer in 2020
Modern Times Beer, who brewed 8 beers in 2019, 2020 and 2021
Pizza Port Brewing Company, who brewed 3 beers in 2020 and 2021
Societe Brewing Company, who brewed 1 beer in 2020
Stone Brewing, who brewed 3 beers in 2019 and 2020
